spec/rubocop/cop/style/nested_parenthesized_calls_spec.rb

```diff
@@ -102,16 +99,16 @@
   end
 
   context 'backslash newline in method call' do
-    let(:source) do
-      <<~RUBY
+    it 'registers an offense' do
+      expect_offense(<<~RUBY)
```

Comment:
Probably should be `<<~'RUBY'`, there's another PR that shows we have issues with these `\`

Reply:
Yes, this is what prompted me to open #8448. I'll happily change it here once that's merged in.